circle.yml

```diff
@@ -8,14 +8,22 @@ machine:
 
```

Comment:
Well the build part, so the whole directory, that you could submit to the chrome store